packages/e2e-tests/experimental-features.js

```diff
@@ -129,4 +129,36 @@ export const siteEditor = {
 
 		await elementToClick.click();
 	},
+
+	async getEditedPostContent() {
```

Comment:
We have this for the post editor but that doesn't work for the site editor. Rather than creating a selector in `edit-site` I opted to create it as a test utility function. Reason: We don't have a use for this selector in the site editor yet so it wouldn't make sense to add it just for E2E tests.